src/components/Sidebar/SidebarNavItem/SidebarNavItem.spec.tsx

```
@@ -0,0 +1,254 @@
+import React from 'react';
+import { describe, it, vi, expect } from 'vitest';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { BrowserRouter } from 'react-router';
+import SidebarNavItem from './SidebarNavItem';
+
+// Mock icon element for testing
+const mockIconElement = <div data-testid="mock-icon" />;
+
+// Mock SVG element for testing
+const mockSvgElement = (
+  <svg data-testid="mock-svg-icon" width="20" height="20">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)